src/view.rs

```diff
@@ -859,3 +859,23 @@ mod view_response_builder_tests {
         assert_eq!(expected, got);
     }
 }
+
+pub struct AllDocumentsViewValue {
```

Comment:
A bit of bike-shedding: I generally don't like types with words like <q>value,</q> <q>data,</q> etc. in their name. These words usually connote nothing useful.

I would prefer the type name `AllDocumentsViewResponse`. It would at least be consistent with other types such as `ErrorResponse` and `ViewResponse` as connoting information that comes back as a response from the CouchDB server.